package.json

```diff
@@ -114,5 +114,8 @@
     "webpack-cli": "^3.3.12",
     "webpack-merge": "^5.1.4",
     "webpack-plugin-serve": "^1.0.1"
+  },
+  "dependencies": {
+    "@edgeware/node-types": "^2.4.5"
```

Comment:
This is actually already available in https://github.com/polkadot-js/apps/blob/master/packages/apps-config/package.json#L16 (local to apps-config so should not need another add, probably may need a bump)

Reply:
Fixed!